scripts/analyze.sh

```diff
@@ -304,7 +304,23 @@ if [ $? -eq 0 ] || check_config_file ZJS_SENSOR; then
     >&2 echo Using module: Sensor
     MODULES+=" -DBUILD_MODULE_SENSOR"
     echo "export ZJS_SENSOR=y" >> zjs.conf.tmp
-    if [ $BOARD = "arduino_101" ]; then
+    if [ $DEV = "ashell" ]; then
```

Comment:
@grgustaf you still want to easily toggle these through zjs.conf, right? so if you put these in the prj.conf.dev, it will always get included.